### PR TITLE
CPP Queues IA Added

### PR DESCRIPTION
**Pull request by ManrajGrover**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New {CPP Queues} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

This pull request adds CPP Queues IA 

###### Which issues (if any) does this fix?

None

###### People to notify (@mention interested parties)

@zekiel @moollaza 
---

Instant Answer Page: https://duck.co/ia/view/cpp_queue_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention
None

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=cpp queue cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/cpp_queue_cheat_sheet_mobile.png?kmr3lWVPD3)

**Comment by edgesince84** [comment]
thanks for the PR @ManrajGrover - we will get this reviewed shortly!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=cpp queue cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/cpp_queue_cheat_sheet_mobile.png?GUXLQVuRfj)

File: share/goodie/cheat_sheets/json/cpp-queue.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
+{
+    "id": "cpp_queue_cheat_sheet",
+    "name": "C++ Queue",
+    "description": "queues are a type of container adaptor, specifically designed to operate in a FIFO context (first-in first-out), where elements are inserted into one end of the container and extracted from the other",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/cpp-queue.json]: Please could this start with an uppercase letter: "Queues"

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=cpp queue cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/cpp_queue_cheat_sheet_mobile.png?G9f0oAxic2)

**Comment by tagawa** [comment]
Thank you for yet another C++ Instant Answer @ManrajGrover! It looks OK on the beta server and we've had no further feedback so we should be good to merge this. 😃

**Comment by daxtheduck** [comment]
@ManrajGrover this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**